users/models.py

```diff
@@ -1,9 +1,29 @@
-from django.utils.timezone import now
+from django.db.models.signals import post_save
 from django.contrib.auth.models import AbstractUser
 from django.db import models
 
 
 class CustomUser(AbstractUser):
+    email = models.EmailField('email address', unique=True, blank=True)
     birthday = models.DateField('%m/%d/%y')
     is_active = models.BooleanField(default=False)
     activate_time = models.DateTimeField(default=None, null=True)
+    link_time = models.DateTimeField(default=None, null=True)
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(CustomUser, on_delete=models.CASCADE,
+                                primary_key=True)
+
+
+class UserGame(models.Model):
+    id = models.IntegerField(primary_key=True)
+    user_profile = models.ForeignKey(UserProfile, on_delete=models.CASCADE, related_name='games')
+
+
+def create_user_profile(sender, instance, created, **kwargs):
+    if created:
+        UserProfile.objects.create(user=instance)
```

Comment:
Why don't you just reference the `CustomUser` model from `UserGame`? Also it is better to use replace `UserGame` with `Favourite` name. The `Favourite` model should contain `game_id` and reference to user. Also does this model belong to `users` application?